### PR TITLE
Add new Professors 1

### PR DESCRIPTION
**Pull request by cseswr**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

_Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

**Inclusion criteria**

- [ ] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [ ] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [ ] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [ ] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ ] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [ ] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -2411,6 +2411,7 @@ Boguslaw Obara,Durham University,https://www.dur.ac.uk/computer.science/staff/pr
 Bohyung Han,Seoul National University,http://cvlab.snu.ac.kr/~bhhan,9aaeCToAAAAJ
 Boi Faltings,EPFL,http://liawww.epfl.ch/~faltings,X21nbgcAAAAJ
 Bojan Cukic,UNC - Charlotte,https://cci.uncc.edu/directory/bojan-cukic,B7N812UAAAAJ
+Bolei Zhou,Chinese University of Hong Kong,http://bzhou.ie.cuhk.edu.hk/,9D4aG8AAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:2414]: Not in a CS department. Please verify inclusion criteria. (I note that you have not checked a single checkbox in the contributions checklist.)

File: csrankings.csv [hunk 2]
```diff
@@ -3758,6 +3759,7 @@ Dag Svanaes,NTNU,https://www.ntnu.edu/employees/dag.svanes,rrAsOFsAAAAJ
 Dag Svanæs,NTNU,https://www.ntnu.edu/employees/dag.svanes,rrAsOFsAAAAJ
 Dag Wedelin,Chalmers/GU,https://www.chalmers.se/en/staff/Pages/dag.aspx,5TbkPtwAAAAJ
 Dagan Feng 0003,University of Sydney,http://sydney.edu.au/engineering/it/about/people/staff/feng.shtml,89py58oAAAAJ
+Dahua Lin,Chinese University of Hong Kong,http://dahualin.org/,GMzzRRUAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:3762]: As above, Department of Information Engineering is not clearly CS.

File: csrankings.csv [hunk 3]
```diff
@@ -19971,6 +19973,7 @@ Xiaofeng Wang 0006,Indiana University,http://www.informatics.indiana.edu/xw7,u4d
 Xiaogang Jin 0001,Zhejiang University,http://www.cad.zju.edu.cn/home/jin,yryOvLwAAAAJ
 Xiaogang Jin 0002,Zhejiang University,http://mypage.zju.edu.cn/0096364,tYqG3CAAAAAJ
 Xiaogang Ma,University of Idaho,http://www.uidaho.edu/engr/departments/cs/our-people/faculty/xiaogang-ma,o4v-pvoAAAAJ
+Xiaogang Wang,Chinese University of Hong Kong,http://www.ee.cuhk.edu.hk/~xgwang/,B5JgjsAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:19976]: None of these folks are listed here, making their eligibility for inclusion unclear: https://www.cse.cuhk.edu.hk/en/people/faculty/by-profile

**Review by emeryberger — CHANGES_REQUESTED** [review]

**Comment by cseswr** [comment]
Dear Emery Berger,

Prof. Xiaogang Wang, Prof. Dahua Lin and Prof. Bolei Zhou are affiliate 
members of Department of Computer Science and Engineering.  They will be 
included in https://www.cse.cuhk.edu.h/en/people/faculty/by-profile once 
we receive the official appointment letter from Human Resources Office 
of The Chinese University of Hong Kong.

Thank you very much for your help.

Best regards,
Fiona Lam
Department of Computer Science and Engineering

**Comment by cseswr** [comment]
Dear Emery Berger,

Prof. Xiaogang Wang, Prof. Dahua Lin and Prof. Bolei Zhou are affiliate 
members of Department of Computer Science and Engineering.  They will be 
included in https://www.cse.cuhk.edu.h/en/people/faculty/by-profile once 
we receive the official appointment letter from Human Resources Office 
of The Chinese University of Hong Kong.

Thank you very much for your help.

Best regards,
Fiona Lam
Department of Computer Science and Engineering

**Comment by cseswr** [comment]
Dear Emery Berger,

Prof. Xiaogang Wang, Prof. Dahua Lin and Prof. Bolei Zhou are affiliate 
members of Department of Computer Science and Engineering.  They will be 
included in https://www.cse.cuhk.edu.h/en/people/faculty/by-profile once 
we receive the official appointment letter from Human Resources Office 
of The Chinese University of Hong Kong.

Thank you very much for your help.

Best regards,
Fiona Lam
Department of Computer Science and Engineering

**Pull request closed without merging**